src/modules/0.0.3/aws_acm_list/index.ts

```diff
@@ -0,0 +1,72 @@
+import { AWS, } from '../../../services/gateways/aws'
+import { Context, Crud, Mapper, Module, } from '../../interfaces'
+import { Certificate } from './entity'
+import * as metadata from './module.json'
+
+export const AwsAcmListModule: Module = new Module({
+  ...metadata,
+  utils: {
+    certificateMapper: (e: any) => {
+      const out = new Certificate();
+      if (!e?.CertificateArn) throw new Error('No CertificateArn defined');
+      out.arn = e.CertificateArn;
+      out.certificateId = e.CertificateArn.split('/').pop();
+      out.certificateType = e.Type;
+      out.domainName = e.DomainName;
+      out.inUse = !!e.InUseBy?.length;
+      out.renewalEligibility = e.RenewalEligibility;
+      out.status = e.Status;
+      return out;
+    },
+  },
+  mappers: {
+    certificate: new Mapper<Certificate>({
+      entity: Certificate,
+      entityId: (e: Certificate) => e.arn ?? e.id.toString(),
+      equals: (a: Certificate, b: Certificate) => Object.is(a.certificateId, b.certificateId) &&
+        Object.is(a.certificateType, b.certificateType) &&
+        Object.is(a.domainName, b.domainName) &&
+        Object.is(a.inUse, b.inUse) &&
+        Object.is(a.renewalEligibility, b.renewalEligibility) &&
+        Object.is(a.status, b.status),
+      source: 'db', // TODO: source cloud??
```

Comment:
Yeah, why not set it like that right now?

Reply:
hmm because I'm actually going to remove that comment and leave it as `source: 'db'`. Because from this table you **can** delete the cloud resources if you delete a row, so is still being manage by the `db`.

Reply:
Ah, got it.